pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -5,47 +5,53 @@ task qiime_import_from_bam {
         description: "Parsing demultiplexed fastq BAM files into qiime readable files."
     }
     input { 
-        File    reads_bam
-        String  sample_name
-        Int     memory_mb = 2000
-        Int     cpu = 1
-        Int     disk_size_gb = ceil(2*size(reads_bam, "GiB")) + 5
+        Array[File] reads_bam
+        Int     memory_mb = 7000
+        Int     cpu = 5
+        Int     disk_size_gb = ceil(2*20) + 5
         String  docker     = "quay.io/broadinstitute/qiime2:conda" 
```

Comment:
Can you do one more thing and go through the whole `tasks_16S_amplicon.wdl` file and replace all these docker strings with the simpler docker string below (without the `:conda`)?

```suggestion
        String  docker     = "quay.io/broadinstitute/qiime2" 
```

Reply:
Addressed in VS